packages/web-react/src/components/Grid/demo/GridItemFactory.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+type GridItemFactoryProps = {
+  items: number;
+  label: string;
+  classes?: string;
+};
+
+const GridItemFactory = ({ items, label, classes }: GridItemFactoryProps) => {
```

Comment:
I am not sure if the `GridItemFactory` fits here when we use it only for one demo. Maybe we can refactor it to be more general to accommodate all Grid demos.

```jsx
const gridItems = [
  { over: 4, label: 'test' },
  // ...
];

<GridItemFactory items={gridItems} />
```

Internally it should use the `DocsBox` component so we do not need to take care of the className.